src/SFA.DAS.EmployerIncentives.Domain.UnitTests/ApprenticeshipIncentive/WhenCalculateEarnings.cs

```diff
@@ -34,8 +34,8 @@ public void Arrange()
         {
             _fixture = new Fixture();
 
-            _plannedStartDate = DateTime.Now.AddDays(5);
-            _collectionPeriod = DateTime.Now;
+            _collectionPeriod = new DateTime(2021, 1, 1);
```

Comment:
We shouldn't be using fixed dates.  The logic should work based on relative tests.